src/main/java/pvs/app/service/ProjectService.java

```diff
@@ -126,4 +139,22 @@ public boolean addGitlabRepo(AddGitLabRepositoryDTO addGitlabRepositoryDTO) thro
         projectDAO.save(project);
         return true;
     }
+
+    public boolean addTrelloBoard(AddTrelloBoardDTO addTrelloBoardDTO) {
+        Optional<Project> projectOptional = projectDAO.findById(addTrelloBoardDTO.getProjectId());
+        if (projectOptional.isEmpty()) return false;
+
+        Project project = projectOptional.get();
+        String url = addTrelloBoardDTO.getRepositoryURL();
+        Repository repository = new Repository();
+        repository.setUrl(url);
+        repository.setType("trello");
+        project.getRepositorySet().add(repository);
+        String responseURL = trelloApiService.getAvatarURL();
+        if (null != responseURL) {
```

Comment:
Suggest change to `expected <-> actual` pattern in every condition statements.
```suggestion
        if (responseURL != null) {
```